src/bp/core/bot-loader.ts

```diff
@@ -17,22 +15,24 @@ import { TYPES } from './types'
 
 @injectable()
 export class BotLoader {
+  public mountBot: Function
```

Comment:
If you do it here, you also share signature
![image](https://user-images.githubusercontent.com/1315508/50903244-40634280-13eb-11e9-9382-29e10c7a21f2.png)

Reply:
@rndlaine